### PR TITLE
Eliminate swagger-specific issues around AutoREST:

### PR DESCRIPTION
**Pull request by akak1977**
1. Added prefix generation to action operation_ids (to allow AutoREST correctly group endpoints accordingly to server controllers);
2. Platform swagger.json returns only platform endpoints;
3. Swagger UI correctly shows all endpoints;
4. Some operation filters thrown out.

### CODE REVIEW

File: src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -83,9 +87,12 @@ public static void AddSwagger(this IServiceCollection services)
 
                 c.DocInclusionPredicate((docName, apiDesc) =>
                 {
-                    if (docName.EqualsInvariant(platformDocName)) return true;
+                    if (docName.EqualsInvariant(platformUIDocName)) return true; // It's an UI endpoint, return all to correctly build swagger UI page
 
-                    var currentAssembly = ((ControllerActionDescriptor)apiDesc.ActionDescriptor).ControllerTypeInfo.Assembly;
+                    if (((ControllerActionDescriptor)apiDesc.ActionDescriptor).ControllerTypeInfo.FullName.StartsWith(platformDocName)) return true; // It's a Platform endpoint. Return platform only
```
Comment (kostyrin) [inline src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs]: - add comparing docName
- better use AssemblyName comparing

File: src/VirtoCommerce.Platform.Web/Swagger/ModuleTagsFilter.cs [hunk 2]
```diff
@@ -1,20 +0,0 @@
-using Swashbuckle.AspNetCore.Swagger;
```
Comment (kostyrin) [inline src/VirtoCommerce.Platform.Web/Swagger/ModuleTagsFilter.cs]: have to return

File: src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs [hunk 3]
```diff
@@ -69,10 +72,11 @@ public static void AddSwagger(this IServiceCollection services)
                 c.OperationFilter<FileUploadOperationFilter>();
                 c.OperationFilter<SecurityRequirementsOperationFilter>();
                 c.OperationFilter<TagsFilter>();
-                c.DocumentFilter<TagsFilter>();
```
Comment (kostyrin) [inline src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs]: have to return

File: src/VirtoCommerce.Platform.Web/Swagger/TagsFilter.cs [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
 namespace VirtoCommerce.Platform.Web.Swagger
 {
-    public class TagsFilter : IDocumentFilter, IOperationFilter
```
Comment (kostyrin) [inline src/VirtoCommerce.Platform.Web/Swagger/TagsFilter.cs]: have to return

File: src/VirtoCommerce.Platform.Web/Swagger/TagsFilter.cs [hunk 5]
```diff
@@ -18,27 +18,6 @@ public TagsFilter(IModuleCatalog moduleCatalog, ISettingsManager settingManager)
             _settingManager = settingManager;
         }
 
-
```
Comment (kostyrin) [inline src/VirtoCommerce.Platform.Web/Swagger/TagsFilter.cs]: have to return

File: src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs [hunk 6]
```diff
@@ -83,9 +87,12 @@ public static void AddSwagger(this IServiceCollection services)
 
                 c.DocInclusionPredicate((docName, apiDesc) =>
                 {
-                    if (docName.EqualsInvariant(platformDocName)) return true;
+                    if (docName.EqualsInvariant(platformUIDocName)) return true; // It's an UI endpoint, return all to correctly build swagger UI page
 
-                    var currentAssembly = ((ControllerActionDescriptor)apiDesc.ActionDescriptor).ControllerTypeInfo.Assembly;
+                    if (((ControllerActionDescriptor)apiDesc.ActionDescriptor).ControllerTypeInfo.FullName.StartsWith(docName)) return true; // It's a Platform endpoint. Return platform only
```
Comment (kostyrin) [inline src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs]: comparing assembly

File: src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs [hunk 7]
```diff
@@ -102,7 +109,7 @@ public static void AddSwagger(this IServiceCollection services)
         public static void UseSwagger(this IApplicationBuilder applicationBuilder)
         {
             applicationBuilder.UseSwagger(c =>
-            {
+            {
```
Comment (artem-dudarev) [inline src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs]: Whitespaces at the end of the line are not allowed.

Automatically format documents when saving:
Install https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.PowerCommandsforVisualStudio
Go to `Tools > Options > Productivity Power Tools > PowerCommands`
Enable `Format documents on save` and `Remove and Sort Usings on save`

File: src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs (hunk 1 above)
Comment (akak1977) [reply src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs]: OK. I did it in another way.

File: src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs (hunk 6 above)
Comment (akak1977) [reply src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs]: OK

File: src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs (hunk 7 above)
Comment (akak1977) [reply src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs]: Power commands was run thru

File: src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs [hunk 8]
```diff
@@ -83,15 +86,15 @@ public static void AddSwagger(this IServiceCollection services)
 
                 c.DocInclusionPredicate((docName, apiDesc) =>
                 {
-                    if (docName.EqualsInvariant(platformDocName)) return true;
+                    if (docName.EqualsInvariant(platformUIDocName)) return true; // It's an UI endpoint, return all to correctly build swagger UI page
 
                     var currentAssembly = ((ControllerActionDescriptor)apiDesc.ActionDescriptor).ControllerTypeInfo.Assembly;
+                    if (docName == platformDocName && currentAssembly.FullName.StartsWith(docName) ) return true; // It's a platform endpoint.
```
Comment (kostyrin) [inline src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs]: use EqualInvariant

Comment (akak1977) [reply src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs]: Fixed

**Pull request merged**